hadoop-hdds/container-service/pom.xml

```diff
@@ -228,6 +228,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>hdds-test-utils</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jul-to-slf4j</artifactId>
+    </dependency>
```

Comment:
nit: please add it above `<!-- Test dependencies -->` line, preferably among other `org.slf4j` dependencies.